falcon/response.py

```diff
@@ -417,6 +524,12 @@ def _wsgi_headers(self, media_type=None):
         if six.PY2:  # pragma: no cover
             # PERF(kgriffs): Don't create an extra list object if
             # it isn't needed.
-            return headers.items()
+            items = headers.items()
+        else:
+            items = list(headers.items())  # pragma: no cover
+
+        if self._cookies is not None:
+            items += [("set-cookie", c.OutputString())
```

Comment:
Might be worth adding a comment here about how this approach executes more quickly than `self._cookies.output().split('\r\n')`.